src/components/Modal.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import ReactDom from 'react-dom';
+import ModalCSS from './ModalCSS.module.css';
```

Comment:
Can you rename this to styles for consistency?

Reply:
I agree with this.
You can change it as following:
import styles from 'ModalCSS.module.css